components/webfield/ExpertiseConsole.js

```diff
@@ -21,6 +22,7 @@ export default function ExpertiseConsole({ appContext }) {
   } = useContext(WebFieldContext)
   const [shouldReload, reload] = useReducer((p) => !p, true)
   const router = useRouter()
+  const useControllableExpertiseSelector = true
```

Comment:
this value must be a parameter of the ExpertiseConsole and it should be false by default.

Reply:
This change is missing, could you please add it so we can merge this soon?

Reply:
Thanks Melisa. Refactored it to be a parameter which can be passed from [here](https://github.com/RonLek/openreview-web/blob/feat/expertise-ui/pages/invitation/index.js#L92). 

Please let me know if you'd like this to be a part of the `webComponentProps` instead, although I'm not sure how [componentObj](https://github.com/RonLek/openreview-web/blob/feat/expertise-ui/pages/invitation/index.js#L64) should be modified only for ExpertiseConsole while still keeping the code generalized.